src/createAnimatedComponent/InlinePropManager.ts

```diff
@@ -132,14 +132,15 @@ export class InlinePropManager implements IInlinePropManager {
   _inlinePropsViewDescriptors: ViewDescriptorsSet | null = null;
   _inlinePropsMapperId: number | null = null;
   _inlineProps: StyleProps = {};
+  private _component: AnimatedComponent;
```

Comment:
Why this is called `_component` and in JSPropUpdater we call it `_animatedComponent`? Both are animated, aren't they?